analyses/sex-prediction-from-RNASeq/run-sex-prediction-from-RNASeq.sh

```diff
@@ -62,59 +90,76 @@ Rscript --vanilla 01-clean_split_data.R \
   --test_targets_file_name $TEST_TARGETS_FILE_NAME \
   --full_targets_file_name $FULL_TARGETS_FILE_NAME \
   --seed $SEED \
-  --train_percent $TRAIN_PERCENT
+  --train_percent $TRAIN_PERCENT \
+  --target_columns $targetColumns_to_pass
 
-# argument for script 02 processing
-# this specifies what column in the target data frame will be used as labels
-# during training
-TRAIN_TARGET_COLUMN=reported_gender
+
+
+for (( i = 0; i < ${#TRANSCRIPT_TAIL_PERCENT_ARRAY[*]}; i++ )); do
 
 # output files for script 02
 # re: the elasticnet model
-MODEL_OBJECT_FILE_NAME=${FILENAME_LEAD}_${SEED}_${TRANSCRIPT_TAIL_PERCENT}_model_object.RDS
-MODEL_TRANSCRIPTS_FILE_NAME=${FILENAME_LEAD}_${SEED}_${TRANSCRIPT_TAIL_PERCENT}_model_transcripts.RDS
-MODEL_COEFS_FILE_NAME=${FILENAME_LEAD}_${SEED}_${TRANSCRIPT_TAIL_PERCENT}_model_coefs.tsv
+MODEL_OBJECT_FILE_NAME=${FILENAME_LEAD}_${SEED}_${TRANSCRIPT_TAIL_PERCENT_ARRAY[i]}_model_object.RDS
```

Comment:
Can you indent everything in this `for` loop to make it clear that it is in a loop
```suggestion
  MODEL_OBJECT_FILE_NAME=${FILENAME_LEAD}_${SEED}_${TRANSCRIPT_TAIL_PERCENT_ARRAY[i]}_model_object.RDS
```

Reply:
Will do all lines on my local repositiory and push.